pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -158,22 +159,26 @@ func (gb *GraphBuilder) controllerFor(resource schema.GroupVersionResource, kind
 }
 
 func (gb *GraphBuilder) monitorsForResources(resources map[schema.GroupVersionResource]struct{}) error {
+	var errors []error
```

Comment:
Based on my read of the code, we will never allow this method to fail.  I think I'd rather inline the `utilruntime.HandleError` right here.